src/components/PDFView/index.js

```diff
@@ -238,9 +240,10 @@ class PDFView extends Component {
 
         // If we're requesting a password then we need to hide - but still render -
         // the PDF component.
-        const pdfContainerStyle = this.state.shouldRequestPassword
-            ? [styles.PDFView, styles.noSelect, this.props.style, styles.invisible]
-            : [styles.PDFView, styles.noSelect, this.props.style];
+        const pdfContainerStyle =
```

Comment:
Let's simplify this. We can assign `[styles.PDFView, styles.noSelect, this.props.style]` by default and `push` additional style `styles.invisible` if it is required.

Reply:
Totally agreed, resolved in 67ab4a531d65c6b777de1e6b54050101bc0d16d2